stdlib/public/Reflection/TypeLowering.cpp

```diff
@@ -1395,13 +1399,13 @@ TypeConverter::getThinFunctionTypeInfo() {
 /// context pointer. The context is modeled exactly like a native Swift
 /// class reference.
 const TypeInfo *
-TypeConverter::getThickFunctionTypeInfo() {
+TypeConverter::getThickFunctionTypeInfo(remote::MemoryReader &reader) {
   if (ThickFunctionTI != nullptr)
     return ThickFunctionTI;
 
   RecordTypeInfoBuilder builder(*this, RecordKind::ThickFunction);
-  builder.addField("function", getThinFunctionTypeRef());
-  builder.addField("context", getNativeObjectTypeRef());
+  builder.addField("function", getThinFunctionTypeRef(), reader);
```

Comment:
This shouldn't need a reader. A thick function is just a pair of pointers as far as RemoteMirrors are concerned.